pkgs/applications/accessibility/dasher/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, lib, fetchFromGitHub
+, autoreconfHook, pkgconfig, wrapGAppsHook
+, gnome3, glib, intltool, gtk3, gnome-doc-utils
+, at-spi2-core, dbus
+, libxslt, libxml2
+, speechSupport ? true, speechd ? null
+}:
+
+assert speechSupport -> speechd != null;
+
+stdenv.mkDerivation {
+  pname = "dasher";
+  version = "2018-04-03";
+
+  src = fetchFromGitHub {
+    owner = "dasher-project";
+    repo = "dasher";
+    rev = "9ab12462e51d17a38c0ddc7f7ffe1cb5fe83b627";
+    sha256 = "1r9xn966nx3pv2bidd6i3pxmprvlw6insnsb38zabmac609h9d9s";
+  };
+
+  prePatch = ''
+    # tries to invoke git for something, probably fetching the ref
+    echo "true" > build-aux/mkversion
+  '';
+
+  configureFlags = lib.optional (!speechSupport) "--disable-speech";
+
+  nativeBuildInputs = [
+    autoreconfHook
+    wrapGAppsHook
+    pkgconfig
+    # doc generation
+    gnome-doc-utils
```

Comment:
there's this awesome pr that migrates to using `yelp-tools` https://github.com/dasher-project/dasher/pull/169 (gnome-doc-utils is deprecated).
Not sure if that branch is up-to-date, but fedora has it as a downstream patch https://src.fedoraproject.org/rpms/dasher/blob/master/f/gnome-doc-utils-depr.patch. I think we should use this.

Reply:
Let’s move when they merge that.

Reply:
I commented on the PR if they could with any updates.